nixos/modules/services/monitoring/scrutiny.nix

```diff
@@ -191,8 +199,10 @@ in
         '';
         serviceConfig = {
           DynamicUser = true;
-          ExecStart = "${getExe cfg.package} start --config ${settingsFormat.generate "scrutiny.yaml" cfg.settings}";
+          ExecStart = "${getExe cfg.package} start --config /etc/scrutiny/config.yaml";
```

Comment:
Adding files to some global place, such as /etc/, is bad because a) it makes running multiple instances of a service harder and b) it adds "state".

Here is how `sing-box` does:
- https://github.com/NixOS/nixpkgs/blob/262d03879b9020ea2c3879492e7b5d3ace7addfb/nixos/modules/services/networking/sing-box.nix#L65
- https://github.com/NixOS/nixpkgs/blob/262d03879b9020ea2c3879492e7b5d3ace7addfb/nixos/modules/services/networking/sing-box.nix#L69-L70